infer/src/base/DBWriter.ml

```diff
@@ -385,7 +385,10 @@ module Server = struct
         send Command.Handshake
 end
 
-let use_daemon = Config.((not (buck || genrule_mode)) && jobs > 1)
+let use_daemon =
+  Config.((not (buck || genrule_mode)) && jobs > 1)
+  && match Version.build_platform with Linux | Darwin -> true | Windows -> false
```

Comment:
Done in 199f707495d8be06a38d5723660abd375a53be04 :)

Reply:
Thanks :sweat_smile: I will rebase in the not too far future, and adapt the code at this moment.